src/components/NcSettingsSection/NcSettingsSection.vue

```diff
@@ -164,7 +164,7 @@ $maxWidth: 900px;
 	&__desc {
 		margin-top: -.2em;
 		margin-bottom: 1em;
-		opacity: $opacity_normal;
+		color: var(--color-text-lighter);
```

Comment:
This variable is deprecated and will shortly removed, use its replacement instead:
```suggestion
		color: var(--color-text-maxcontrast);
```